vm/vm.proto

```diff
@@ -27,12 +27,12 @@ message CallTx {
     VMType vm_type = 1;
     bytes input = 2;
     BigUInt value = 3;
-
     string contract_version = 4;
 }
 
 message MigrationTx {
     uint32 id = 1 [(gogoproto.customname)="ID"];
+    bytes migration_params = 2 [(gogoproto.customname)="MigrationParams"];
```

Comment:
`migration_params` -> `input`
And get rid of the `customname` tag, it's not needed here

Reply:
Done.